### ISSUE TITLE
Harfbuzz failure

### ISSUE THREAD

**Issue by shawmat** [mapnik/mapnik#3642]
Mapnik build fails with a Harfbuzz version test, even though this is an optional dependency that I haven't included.  My setup is Ubuntu 16.04, Boost 1.63, GCC6.2 and Python 2.7.12. Memory is 6Gb. Mapnik was cloned from git today (19th March 2017).

```
In file included from src/text/text_layout.cpp:29:0:
include/mapnik/text/harfbuzz_shaper.hpp:120:48: error: missing binary operator before token "("
 #if (HB_VERSION_MAJOR > 0 && HB_VERSION_ATLEAST(1, 0, 5))
                                                ^
scons: *** [src/text/text_layout.os] Error 1
scons: building terminated because of errors.
Makefile:53: recipe for target 'mapnik' failed
make: *** [mapnik] Error 2

```
./configure passes all tests:
[mapnik_config_20170319.txt](https://github.com/mapnik/mapnik/files/853139/mapnik_config_20170319.txt)

**Comment by lightmare** [comment]
You're right, this problem is due to https://github.com/mapnik/mapnik/commit/755d178d20ec12b8afe8bcac87f77a7fe0356c3a#diff-fa96dbabf71b3691390d1f69e56df864L118

The reason these were split into two tests is because `HB_VERSION_ATLEAST` didn't exist in earlier versions, so its use must be protected separately.

**Comment by artemp** [comment]
@shawmat - should be fixed in 5450255, pls re-open if not. Thanks @lightmare.
